examples/jsm/loaders/3DMLoader.js

```diff
@@ -714,6 +717,9 @@ class Rhino3dmLoader extends Loader {
 					const message = e.data;
 
 					switch ( message.type ) {
+						case 'warning':
+							worker._callbacks[ message.id ].warn( message );
```

Comment:
We need a id to figure out which task this warning belongs to. I'm using the taskID corresponding to the file to collate the warnings.

Reply:
I am not sure we should add a .warn to this callback, since the promises api only defines `resolve` and `reject`. I think this is the place we should be collecting the warning, or printing them to the console, or doing something. The worker should need to be bothered by a warning unless I am confused.